kmip20/op_setattribute.go

```diff
@@ -0,0 +1,46 @@
+package kmip20
+
+import (
+	"context"
+
+	"github.com/gemalto/kmip-go"
+)
+
+// 6.1.47 Set Attribute
+
+// Table 296
+
+type SetAttributeRequestPayload struct {
+	UniqueIdentifier UniqueIdentifierValue
+	AttributeName    string
+	AttributeValue   string
```

Comment:
I don't think the request payload is supposed to directly contain AttributeName and AttributeValue.  From the spec (https://docs.oasis-open.org/kmip/kmip-spec/v2.0/os/kmip-spec-v2.0-os.html#_Toc6497573) and the test cases, it looks like it should be a structure (defined here: https://docs.oasis-open.org/kmip/kmip-spec/v2.0/os/kmip-spec-v2.0-os.html#_Toc6497524) which can contain any arbitrary attribute.  And from the test cases, it looks like the attributes can either directly be the attribute as a TTLV, or can be an `Attribute` structure.

So maybe this field can be defined like this:

```
    NewAttribute      Attributes  `ttlv:"DerivationData"`
```

Reply:
Thank you. I will review and make changes for the commit and review.